### PR TITLE
(580) Add family accommodation

### PR DESCRIPTION
**Pull request by libuk**
# Context

# Changes in this PR
* Update question copy for the family accommodation page
* Adds a new page to the apply journey FamilyAccommodation

### Before
Screenshot 2023-11-28 at 16 33 37

### After
Screenshot 2023-11-28 at 15 46 31

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Are any changes required to the e2e tests?
- [ ] If you've added a new route, have you added a new
  `auditEvent`? (see `server/routes/apply.ts` for examples)
- [ ] Are there environment variables or other infrastructure configuration which needs to be included in this release?
- [ ] Are there any data migrations required. Automatic or manual?
- [ ] Does this rely on changes being deployed to the CAS API?

## Post merge checklist

Once we've merged it will be auto-deployed to the dev environment.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui).

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

**Comment by libuk** [comment]
[Area - Family accommodation](https://trello.com/c/8JM1TQC5/580-area-family-accommodation)

File: integration_tests/pages/apply/taskListPage.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default class TaskListPage extends Page {
   }
 
   shouldShowTaskStatus = (task: string, status: string): void => {
-    cy.get(`#${task}-status`).should('contain', status)
+    cy.get(`#${task}-status`).debug().should('contain', status)
```
Comment (Harriethw) [inline integration_tests/pages/apply/taskListPage.ts:35]: is this a stray `debug` left in?

File: integration_tests/tests/apply/area_and_funding/area_information/second_preferred_area.cy.ts [hunk 2]
```diff
@@ -85,10 +85,7 @@ context('Visit "Second preferred area" page', () => {
     // When I continue to the next task / page
     page.clickSubmit()
 
-    // Then I am returned to the task list
-    const taskListPage = Page.verifyOnPage(TaskListPage)
-
-    // And I see that the area information task is complete
-    taskListPage.shouldShowTaskStatus('area-information', 'Completed')
+    // Then I am returned to the "Second preferred area" page
```
Comment (Harriethw) [inline integration_tests/tests/apply/area_and_funding/area_information/second_preferred_area.cy.ts:88]:
```suggestion
    // Then I am taken to the "Family accommodation" page
```

**Review by Harriethw — APPROVED** [review]
LGTM! Thanks for the clear PR 👍

File: integration_tests/pages/apply/taskListPage.ts (hunk 1 above)
Comment (libuk) [reply integration_tests/pages/apply/taskListPage.ts:35]: Great catch!

**Pull request merged**